vector/src/main/java/im/vector/app/features/home/room/list/RoomListSectionBuilderSpace.kt

```diff
@@ -445,7 +445,7 @@ class RoomListSectionBuilderSpace(
         return when (spaceFilter) {
             RoomListViewModel.SpaceFilterStrategy.ORPHANS_IF_SPACE_NULL -> {
                 copy(
-                        spaceFilter = currentSpace?.toActiveSpaceOrOrphanRooms()
+                        spaceFilter = currentSpace.toActiveSpaceOrOrphanRooms()
```

Comment:
there is a third instance in the codebase, do we want to update it as well?

Reply:
I only saw 2 :/

Reply:
https://github.com/vector-im/element-android/blob/develop/vector/src/main/java/im/vector/app/features/home/HomeDetailViewModel.kt#L293